Pose current_cell
---
Pose[] global_plan